packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -116,14 +114,14 @@ const Onboarding = () => {
             paddingBottom={2}
             paddingTop={2}
           >
-            {staticLinks.map((link) => (
+            {STATIC_LINKS.map((link) => (
               <StyledLink
```

Comment:
StyledLink might need a little fix, I think we've lost the icon color and hover color 👀
<img width="216" alt="image" src="https://user-images.githubusercontent.com/71838159/208699295-f61c71c7-269b-489d-803c-6b4e23f28b24.png">

Reply:
Addressed it in 6ecb4da5b79cb9a3cf2d66fe1a7a21b6e42736cc